main.go

```diff
@@ -547,7 +548,8 @@ is available in the $BITRISE_XCODE_RAW_RESULT_TEXT_PATH environment variable`)
 				if exportMethod == exportoptions.MethodAppStore {
 					iCloudContainerEnvironment = "Production"
 				} else if cfg.ICloudContainerEnvironment == "" {
-					fail("project uses CloudKit, but iCloud container environment input not specified")
+					fail("Your xcode project uses CloudKit capability and the export method specified is not app-store.\n" +
```

Comment:
"iCloud container environment input not specified" would include the root cause of the error like this:

Your project uses CloudKit and iCloud container environment input not specified. Export method is: %s (For app-store export method Production container environment is implied.)